modules/custom-domains/main.tf

```diff
@@ -1,5 +1,5 @@
 
-module "primary-region" {
+module "primary_region" {
   source = "./modules/certificate-and-domain"
```

Comment:
Should the names of the two module files also get the same conversion?

Reply:
(Discussed elsewhere; decided not to bother at this point, since we use hyphens in folder names elsewhere, such as 000-core, etc.)